chatbot/views.py

```
@@ -0,0 +1,28 @@
+from django.http import JsonResponse
+
+# Simple chatbot responses
+responses = {
+    "hello": "Hi there! How can I assist you?",
+    "how are you": "I'm just a bot, but I'm doing great!",
+    "bye": "Goodbye! Have a great day!",
+    "default": "Sorry, I didn't understand that. Can you rephrase?"
+}
+
+# Function to get chatbot response
+def get_bot_response(user_input):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_bot_response`

(ANN201)